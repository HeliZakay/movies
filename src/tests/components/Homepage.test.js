import React from "react";
import { shallow } from "enzyme";
import Homepage from "../../components/Homepage";

test("should render Homepage correctly", () => {
    const wrapper = shallow(<Homepage />);
    expect(wrapper).toMatchSnapshot();
});