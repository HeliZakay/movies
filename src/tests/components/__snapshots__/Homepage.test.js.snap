// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Homepage correctly 1`] = `
<div>
  <h2>
    Movies recommended by friends
  </h2>
  <p>
    Take a look at your friends' recommendations!
  </p>
  <hr />
  <[object Object] />
  <[object Object] />
</div>
`;
